<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClassClass</class>
 <widget class="QMainWindow" name="MyClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主界面</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>address.svg</normaloff>address.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>file</string>
    </property>
    <widget class="QMenu" name="menuAdd_data">
     <property name="title">
      <string>add data</string>
     </property>
     <addaction name="actionGeojson"/>
     <addaction name="actionShp"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuAdd_data"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionAbout_us"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>frame</string>
    </property>
    <addaction name="actionContent"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>database</string>
    </property>
    <addaction name="actionSave_shapefile"/>
    <addaction name="actionGet_databaseData"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuDatabase"/>
   <addaction name="menu_2"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionGeojson">
   <property name="text">
    <string>geojson</string>
   </property>
  </action>
  <action name="actionShp">
   <property name="text">
    <string>shp</string>
   </property>
  </action>
  <action name="actionAbout_us">
   <property name="text">
    <string>about us</string>
   </property>
  </action>
  <action name="actionContent">
   <property name="text">
    <string>content</string>
   </property>
  </action>
  <action name="actionSave_shapefile">
   <property name="text">
    <string>save shapefile</string>
   </property>
  </action>
  <action name="actionGet_databaseData">
   <property name="text">
    <string>get databaseData</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myclass.qrc"/>
 </resources>
 <connections/>
</ui>
